tests/app/upload/test_rest.py

```diff
@@ -0,0 +1,180 @@
+from datetime import datetime, timedelta, date
+
+from freezegun import freeze_time
+
+from app.models import LETTER_TYPE, JOB_STATUS_FINISHED, JOB_STATUS_PENDING
+from tests.app.db import create_job, create_notification, create_template, create_ft_notification_status
+from tests.conftest import set_config
+
+
+def create_uploaded_letter(letter_template, service, status='created', created_at=None):
+    return create_notification(
+        template=letter_template,
+        to_field="file-name",
+        status=status,
+        reference="dvla-reference",
+        client_reference="file-name",
+        one_off=True,
+        created_by_id=service.users[0].id,
+        created_at=created_at
+    )
+
+
+def create_precompiled_template(service):
+    return create_template(
+        service,
+        template_type=LETTER_TYPE,
+        template_name='Pre-compiled PDF',
+        subject='Pre-compiled PDF',
+        content="",
+        hidden=True,
+        postage="second",
+    )
+
+
+def test_get_uploads(admin_request, sample_template):
+    letter_template = create_precompiled_template(sample_template.service)
+
+    upload_1 = create_uploaded_letter(letter_template, sample_template.service, status='delivered',
+                                      created_at=datetime.utcnow() - timedelta(minutes=4))
```

Comment:
Optional: Technically we don't need the `timedelta` part on all of our timestamps in this test. They would still come out in the same order anyway.